java/src/main/java/ai/rapids/cudf/Aggregation.java

```diff
@@ -56,10 +56,12 @@
         ROW_NUMBER(17),
         COLLECT_LIST(18),
         COLLECT_SET(19),
-        LEAD(20),
```

Comment:
@revans2 I assume that your problem is due to this.

Reply:
No my problem is that @sperlingxx didn't even run the unit tests or how would have found this and there would have been no issue.  I will post a patch shortly

Reply:
Here we arbitrarily define the value for aggregation enums, which is different from the enums defined in libcudf. I was talking about this before (probably with Jason). We should have some better way to automate this. Otherwise, we will continue to have issues with this in the future.

Reply:
These just happen to match the values in libcudf.  But in this case we never cast them so it does not have to match.  This is the one place that we do it correctly.  If we did cast them, then your patch that changed the values would have broken the java build, not this patch.

Reply:
Oh, it's my mistake! I apologized for my carelessness.